mne/io/meas_info.py

```diff
@@ -1211,3 +1214,141 @@ def _empty_info():
+    class directly if you're constructing a new montage.
+
+    Parameters
+    ----------
+    hsp : array, shape (n_points, 3)
+        The positions of the channels in 3d.
+    hpi : array, shape (n_hpi, 3)
+        The positions of the head-position indicator coils in 3d.
+        These points are in the MEG device space.
+    elp : array, shape (n_hpi, 3)
+        The positions of the head-position indicator coils in 3d.
+        This is typically in the acquisition digitizer space.
+    point_names : list, shape (n_elp)
+        The names of the digitized points for hpi and elp.
+    nasion : array, shape (1, 3)
+        The position of the nasion fidicual point in the RAS head space.
+    lpa : array, shape (1, 3)
+        The position of the left periauricular fidicual point in
+        the RAS head space.
+    rpa : array, shape (1, 3)
+        The position of the right periauricular fidicual point in
+        the RAS head space.
+    """
+    def __init__(self, hsp, hpi, elp, point_names,
+                 nasion=None, lpa=None, rpa=None):
+        self.hsp = hsp
+        self.hpi = hpi
+        self.elp = elp
+        self.point_names = point_names
+
+        self.nasion = nasion
+        self.lpa = lpa
+        self.rpa = rpa
+
+    def __repr__(self):
+        s = '<DigMontage | %d Dig Points, %d HPI points: %s ...>'
+        s %= self.kind, len(self.ch_names), ', '.join(self.point_names[:3])
+        return s
+
+    def plot(self, scale_factor=1.5, show_names=False):
+        """Plot EEG sensor montage
+
+        Parameters
+        ----------
+        scale_factor : float
+            Determines the size of the points. Defaults to 1.5
+        show_names : bool
+            Whether to show the channel names. Defaults to False
+
+        Returns
+        -------
+        fig : Instance of matplotlib.figure.Figure
+            The figure object.
+        """
+        from ..viz import plot_montage
+        return plot_montage(self, scale_factor=scale_factor,
+                            show_names=show_names)
+
+
+def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
+                     transform=True):
+    """Read montage from a file
+
+    Note: Digitized points will be transformed to head-based coordinate system.
+
+    Parameters
+    ----------
+    hsp : None | str | array, shape (n_points, 3)
+        If str, this corresponds to the filename of the headshape points.
+        If numpy.array, this corresponds to an array of positions of the
+        channels in 3d.
+    hpi : None | str | numpy.array, shape (n_channels, 3)
+        If str, this corresponds to the filename of hpi points. If numpy.array,
+        this corresponds to an array hpi points. These points are in
+        device space.
+    elp : None | str | numpy.array
+        If str, this corresponds to the filename of hpi points. If numpy.array,
+        this corresponds to an array hpi points. These points are in
```

Comment:
s/hpi/elp. Also, please say what this means, e.g. "electrode points" as you did above for `hsp`